src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
```

Comment:
Add a comment to briefly explain the purpose of the function, it helps other developers to understand the code easily 🙂

Reply:
Done